.github/workflows/first_interaction.yml

```diff
@@ -0,0 +1,44 @@
+name: First interaction
+
+on:
+  - pull_request_target
+  - issues
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+      issues: write
+
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: "${{ secrets.GITHUB_TOKEN }}"
+        issue-message: |
+          Thank you for contributing an issue!
+
+          **We are glad that you are finding DIPY useful!**
+
```

Comment:
I would add a sentence saying to move the issue to discussions if asking for help or advice. issues a dedicated for new features request or bug founded.

Reply:
Done https://github.com/dipy/dipy/pull/3387/commits/ab12ade3cb0ce2d44b4827145d14c507be853a10#diff-0389854d99ec31d393a8a889246f3b9aa55395e692bbfa2a3142b8f28adf5ed0R26.